src/hooks/useHttp.test.js

```diff
@@ -0,0 +1,55 @@
+import { act, renderHook } from '@testing-library/react-hooks';
+import { fetchVideos } from '../lib/api';
+import useHttp from './useHttp';
+import mockData from '../helper/mock-data.json';
+
+describe('useHttp Hook', () => {
+  window.fetch = jest.fn();
+
+  test('fetches videos by url constructed correcty', async () => {
+    const url = `https://www.googleapis.com/youtube/v3/search?part=snippet&key=AIzaSyBfc4oeDp-VCHubNeaFbwuPz-_LKiWMn7E&maxResults=20&q=react&type=video&safeSearch=strict`;
```

Comment:
Just saw that you are using `.env` in the actual project but will leave this comment just in case that is a real API key here